package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
   "exports": {
-    ".": "./lib/main.js",
+    ".": {
+      "default": "./lib/main.js",
```

Comment:
Before I merge this:

can you point me to the definition of using `default` rather than specifying both `import` and `require`? I cannot find the documentation on this.

Reply:
I also wonder if moving our type definition file under lib would solve a lot of these edge cases - from the past and going forward. That was the historical default and most editors I believe are set up to look there.

Reply:
doc of `default`, `import` and `require` is here.
https://nodejs.org/api/packages.html#conditional-exports

I think it will be better if we moving our type definition file under `lib`. Should I create another PR to do this?

Reply:
I agree. Can you just roll it into this PR, please? Thank you for your help!

Reply:
@motdotla see https://github.com/motdotla/dotenv/pull/585